docroot/themes/contrib/civictheme/civictheme_library/components/01-atoms/table/table.stories.js

```diff
@@ -26,25 +26,25 @@ export const Table = (knobTab) => {
 
   const rows = [
     [
-      "<a class='civictheme-link' href='#' title='Row 1 with link'>Row 4 with link</a>",
+      "<a class='civictheme-content-link' href='#' title='Row 1 with link'>Row 4 with link</a>",
```

Comment:
@joshua-salsadigital 

Could you please remove the class from the links completely so Storybook looks like Drupal

Reply:
Updated